src/rocksdb_options.rs

```diff
@@ -949,6 +974,10 @@ impl Options {
             crocksdb_ffi::crocksdb_options_set_allow_concurrent_memtable_write(self.inner, v);
         }
     }
+
+    pub fn set_env(&mut self, env: &Env) {
```

Comment:
It is bad to let the users manage the lifetime of Env. We should only DB outside only.